AndroidSDKCore/src/main/java/com/leanplum/internal/Util.java

```diff
@@ -556,7 +557,8 @@ static HttpURLConnection createHttpUrlConnection(
     urlConnection.setRequestProperty("User-Agent",
         getApplicationName(context) + "/" + getVersionName() + "/" + RequestOld.appId() + "/" +
             Constants.CLIENT + "/" + Constants.LEANPLUM_VERSION + "/" + getSystemName() + "/" +
-            getSystemVersion() + "/" + Constants.LEANPLUM_PACKAGE_IDENTIFIER);
+            getSystemVersion() + "/" + Constants.LEANPLUM_SUPPORTED_ENCODING + "/" + Constants.LEANPLUM_PACKAGE_IDENTIFIER);
+    urlConnection.setRequestProperty("Accept-Encoding", Constants.LEANPLUM_SUPPORTED_ENCODING);
```

Comment:
Could you add a comment here as to why we're appending "gzip" to the end of the user agent?
This is done for compatibility with AppEngine Standard:
1. Must include `Accept-Encoding: gzip` in the heaer
2. Must include the phrase `gzip` in the `User-Agent` header
https://cloud.google.com/appengine/kb/

Reply:
sure